packages/matter-node.js/src/matter/interaction/InteractionServer.ts

```diff
@@ -323,6 +323,9 @@ export class InteractionServer implements ProtocolHandler<MatterDevice> {
         await Promise.all(invokes.map(async ({ path, args }) => {
             const command = this.commands.get(commandPathToId(path));
             if (command === undefined) return;
+            if (args === undefined) { // If no arguments are provided in the invoke data, we use an empty TlvStream
```

Comment:
I am confused: from the modification above, this is never undefined but an empty object instead.
What am I missing?

Reply:
If we have an "optionalField" and this is not provided ... then the value is "undefined", but I think with the above idea of  an own Schema that gets all simplified